JYH/3Day/두개 뽑아서 더하기.js

```diff
@@ -0,0 +1,16 @@
+function solution(numbers) {
+    let answer = [];
+    
+    for (let i = 0; i < numbers.length; i++) {
+        for (let j = i + 1; j < numbers.length; j++) {
+            let sum = numbers[i] + numbers[j];
+            if (answer.indexOf(sum) === -1) {
+                answer.push(sum);
+            }
+        }
+    }
+    answer.sort(function(a, b){
+        return a - b;
+    })
+    return answer;
+}
```

Comment:
```suggestion
answer.sort((a, b) => a - b);
return answer;
```

여기서는 화살표 함수를 이용하여 더 간결하게 작성할 수 있을 것 같네요.👍

아래 링크를 참고해보세용
[MDN-화살표 함수 표현식](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

Reply:
보내주신 링크 잘 보고 더 좋은 방식으로 고치도록 하겠습니다 감사합니다!